shell/packages/sandstorm-db/migrations.js

```diff
@@ -358,6 +358,20 @@ function splitAccountUsersAndIdentityUsers() {
   });
 }
 
+function populateContactsFromApiTokens() {
+  ApiTokens.find({"owner.user": {$exists: 1}}).forEach(function(token) {
+    var identityId = token.owner.user.identityId;
+    var profile = SandstormDb.prototype.getIdentity(identityId).profile;
+    Contacts.upsert({ownerId: token.accountId, identityId: identityId}, {
```

Comment:
Due to a bug that we recently fixed, `token.accountId` might not exist on some tokens.
See [this comment](https://github.com/sandstorm-io/sandstorm/blob/90d86d7d2cc8b7865fb84d17076b10f010ad5069/shell/packages/sandstorm-permissions/permissions.js#L449-L452).
You should either include `{accountId: {$exists: 1}}` in the `ApiTokens` query, or you should repair these tokens.